subscription_oca/models/sale_subscription.py

```diff
@@ -161,6 +163,15 @@ def cron_subscription_management(self):
                     subscription.action_start_subscription()
                     subscription.generate_invoice()
 
+    @api.constrains("template_id")
+    def _constrain_template_id(self):
+        if self.template_id.company_id not in [False, self.env.company]:
```

Comment:
**suggestion:** according to the [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#L902-L903):
```suggestion
    def _check_template_id(self):
        if self.template_id.company_id not in [False, self.env.company]:
```

**question:** have you tested this without selecting any templates? does it still work fine?